pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -359,7 +364,10 @@ private void validateSchemaInternal(Schema schema) {
     validateSchemaName(schema.getSchemaName());
     try {
       List<TableConfig> tableConfigs = _pinotHelixResourceManager.getTableConfigsForSchema(schema.getSchemaName());
-      SchemaUtils.validate(schema, tableConfigs);
+      boolean isIgnoreCase =
+        Boolean.parseBoolean(_controllerConf.getProperty(CommonConstants.Helix.ENABLE_CASE_INSENSITIVE_KEY,
```

Comment:
I think this config is actually read from the ZK (cluster config) instead of controller config. Please take a look at `PinotHelixResourceManager.start()`.
We can probably directly inject the `TableCache` or case-insensitive boolean

Reply:
Thanks for your review and your suggestion. @Jackie-Jiang 
I have changed to ingest directly from TableCache other than controller config. Please take a second look.

Reply:
:+1: sounds